_posts/2017-06-23-release-0.3p5.md

```diff
@@ -0,0 +1,66 @@
+---
+layout: post
+title: "Eve Alpha v0.3 preview 5"
+author: "Corey Montella"
+tags: []
+---
+
+_(Eve is a new programming language, and this is our development blog. If you’re new to Eve, [start here](http://witheve.com))_
+
+We're releasing Preview 5 of Eve v0.3 today. This isn't a huge release, but it contains some critical bug fixes, so if you're using Preview 4, you'll want this right away.
+
+## What's New?
+
+### Eve Starter
+
+Since more people are using `eve-starter`, we've implemented two development streams so we don't break your work inadvertently.
+
+- [release](https://github.com/witheve/eve-starter/tree/release) - Our npm module will be based of this branch, and we will inform you ahead of time of any breaking changes that we make.
```

Comment:
We don't have an npm module for eve-starter atm, that's `witheve` that has the npm module. We potentially could make one but it's a bit weird since it's meant to be a "make your own thing inside of this" kinda repo. It'd make sense to have an `eve-cli` tool or something though as an npm module